guides/nestor/ISPsetup.md

```diff
@@ -0,0 +1,316 @@
+
+## Prerequisites <a id="preinfo"></a>
+1. If you have not setup your network adapter look here https://goo.gl/vs4cHC.
+2. Just copy the Prerequisites of peoples guides before yours here.
+3. Have your [virtual environment](link-to-guide) configured
+4. Have the [ISP](link-to-guide) gateway running.
+5. Have [pfSense](link-to-guide) running.
+
+## Summary <a id="Summary"></a>
+This tutorial will show you how to setup the ISP server you will need for your lab.
+
+<br>
+
+| Network adapters | Description                             |
+|------------------|---------------------------------        |
+| Adapter 1        | Leave it to NAT                         |
+| Adapter 2        | Change to Host-only (vboxnet2)          |
+
+##### Supporting detail 
+The logical ideal of this tutorial is for you to be able to setup the ISP server for your lab. The ISP lab is the one that allow you to connect your other lab to the internet think at it as your internet provider at home.  
+
+## VirtualBox VM Setup <a id="First"></a>
+
+
+1. Go to [Ubuntu](https://www.ubuntu.com/download/server) and download the LTS server version. 
+
+- Install Ubuntu Server in Virtual Machine
+- On Virtual Box go to top left corner to New to create the virtual box.
+
+![Section2](https://i.imgur.com/QnoKC2F.png) 
+
+- Create the name of this Box recommended name `ISP Ubuntu`
+- In Type choose Linux if is not auto setected
+- In Version choose `Ubuntu (64-bit)`
+
+![Section3](https://i.imgur.com/2JtRAzY.png)
+
+- On the memory section depending to your machine RAM capacity the minimun should be (254MB) but if you have enough memory use (1024MB)
+
+![Section4](https://i.imgur.com/3KinU4J.png)
+
+- Leave the Hard drive section as default which is (Create a virutal hard disk now)
+
+![Section5](https://i.imgur.com/OwYGx2s.png)
+
+- Leave the Hard disk file type to `(VDI)`
+
+![Section5](https://i.imgur.com/XPpGqxm.png)
+
+- Leave storage on physical hard disk to `(Dynamically allocated)`
+
+![Section5](https://i.imgur.com/YGrvBmV.png)
+
+- For this machine the minimun for Ubuntu so it does not give you problem set the File location and size to (20.00GB)
+
+![Section5](https://i.imgur.com/p2jIaGn.png)
+
+- After the Virtual Machine is created now you have to add the ISO that you download at the beginning 
+- Go to Storage section and click on top that it say `[Optical Drive]Empty`
+
+![Section6](https://i.imgur.com/8Rfq6nZ.png)
+
+- Then press `Choose disk image...`
+
+![Section6](https://i.imgur.com/Hbb1y7h.png)
+
+- Find the location where you download your ISO for Ubuntu Server.
+
+![Section6](https://i.imgur.com/JHKaWpk.png)
+
+- On this step make sure you add the network adapters to the server 
+
+- Right click the ISP Ubuntu and click settings
+
+![Section7](https://i.imgur.com/8w6rRfO.png)
+
+- On network adapter one leave it to `NAT`
+- Then go to Network Adapter 2 and change the network adapter to `Host-only Adapter` like the picture bellow.
```

Comment:
Since everyone's virtualbox is different I suggest saying: Then go to the Network Adapter and change the network adapter to `Host-only Adapter` like the picture bellow. Make sure the host only adapter you choose is the one associated to the WAN Environment.